enforcer/src/main/java/org/wso2/micro/gateway/enforcer/util/TLSUtils.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the licence header

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1582/commits/b804e40a426ab4a0c8f06cc8c5d68598fbfc8882